src/servers/ZoneServer2016/zoneserver.ts

```diff
@@ -1880,6 +1876,17 @@ export class ZoneServer2016 extends EventEmitter {
   }
 
   async sendInitData(client: Client) {
+    // Load character before sending sendSelf, as we need to send item definitions for the inventory to the client first. This is the same behaviour as Z1BR
+    await this.fetchCharacterData(client);
+
+    // This packet is only necessary for transitioning between different zones.
```

Comment:
Quick question unrelated, do you know how to switch zones?

Reply:
Yes, it's extensively used in BR